results/server/repo_handler.py

```diff
@@ -0,0 +1,41 @@
+import git
+ 
+class RepoHandler:
+
+    def __init__(self,
+            pages_repo_dir: str,
+            pages_branch: str,
+            git_username: str,
+            git_email: str
+            ) -> None:
+        self.pages_repo_dir = pages_repo_dir
+        self.pages_branch = pages_branch
+        self.git_username = git_username
+        self.git_email = git_email
+        
+    @staticmethod
```

Comment:
This should not be a static method as we depend on instantiation to get information such as git username, git email, etc.